packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss

```diff
@@ -1,22 +1,32 @@
 @import '../../common/common';
 
+$spacingAroundItemButton: 4px;
+
 .root {
-  position: relative;
   width: auto;
 }
 
 .clear {
   clear: both;
 }
 
+.members {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: hidden;
+  margin: -1 * $spacingAroundItemButton;
+
+  > * {
```

Comment:
I think this could cause layout breaks in other apps, no? You're pulling in negative margin here.